lib/notation.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Notation do
+  def notate(:false) do
+    "falseB"
+  end
+
+  def notate(:true) do
+    "trueB"
+  end
+
+  def notate(nil) do
+    "_"
+  end
+
+  def notate(data) do
```

Comment:
is it possible to have multiple notate functions instead of this, which have guards to determine the type of data. E.g:

`def notate(data) when is_list(data)`
`def notate(data) when is_map(data)`
`def notate(data) when is_binary(data)`